pkg/network/node/healthcheck_ovs.go

```diff
@@ -0,0 +1,40 @@
+package node
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/openshift/sdn/pkg/util/ovsclient"
+	utilwait "k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/klog/v2"
+)
+
+const (
+	ovsDialDefaultNetwork = "unix"
+	ovsDialDefaultAddress = "/var/run/openvswitch/db.sock"
+)
+
+func healthCheckOVS() error {
+	klog.Infof("[OVS healthcheck] starting OVS health check")
+	c, err := ovsclient.DialTimeout(ovsDialDefaultNetwork, ovsDialDefaultAddress, time.Minute)
+	if err != nil {
+		return fmt.Errorf("[OVS healthcheck] error connecting to OVS: %v", err)
+	}
+	if err := c.Ping(); err != nil {
+		if cErr := c.Close(); cErr != nil {
+			return fmt.Errorf("[OVS healthcheck] error pinging OVS, err: %v, and closing the connection, err: %v", err, cErr)
+		}
+		return fmt.Errorf("[OVS healthcheck] error pinging OVS, err: %v", err)
+	}
+	go utilwait.Until(func() {
+		klog.Infof("[OVS healthcheck] starting SDN-OVS disconnection go-routine")
+		c.WaitForDisconnect()
+		klog.Errorf("[OVS healthcheck] detected OVS server change, restarting")
+		if err := c.Close(); err != nil {
+			klog.Errorf("[OVS healthcheck] could not close connection, err: %v", err)
```

Comment:
The server already disconnected, so presumably you're going to get an error from `Close`? And we're about to exit so it doesn't really matter whether you close first anyway. You could just remove this.